### PR TITLE
9999407-java-duplicate-code-exact-fixer

### PR DESCRIPTION
**Pull request by codefix-service-user**
62420488-125Java fixer to fix duplicate code exact<br/>CAP issue id: 62420488-125

### CODE REVIEW

**Comment by ganapathy-df** [comment]
https://devgraph-alp.atlassian.net/browse/CC-139454
Automated QE Check Result: FAILED
ValidateAllDuplicateLinesAreExactlyTheSame: SUCCESS 
ValidateDuplicateCodeIsEqualOrMoreThanMinimumDuplicateLines: SUCCESS 
ValidateDuplicateCodeStartLineIsCorrect: SUCCESS 
ValidateDuplicateCodeEndLineIsCorrect: SUCCESS 
ValidateThatDuplicateCodeShouldBeExtracted: SUCCESS 
ValidateOnlyRemovedLinesAreAddedIntoExtractedCodeWithSomeExceptions: SUCCESS 
ValidateAddedCodeHasCorrectAccessModifier: SUCCESS 
ValidateThatCallToExtractedMethodIsSameAtAllPlaces: SUCCESS 
ValidateImportsAreCorrect: FAILED 
<pre>Unnecessary added Import(s):
import org.jvnet.hudson.test.JenkinsRule.WebClient;
</pre>
ValidateExtractedMethodIsInCorrectLocation: SUCCESS 
ValidateFilesModifiedInPrMatchesTheFileInJira: SUCCESS 
ValidateMethodParameterOnExtractedCodeIsProperlyUsed - : SUCCESS 
<pre>Parameters: READONLY</pre>
ValidateMethodParameterAndArgumentsAreSame: SUCCESS 
ValidateThatDeletedLinesMatchesTheDuplicateCodesInJira: SUCCESS 
validateHandlingOfVariableAssignmentIsCorrect: SUCCESS 
<pre>List of variables assigned with new values: 
Local Variables: [i, wc]</pre>
ValidateThatAllDeletedLinesAreTheSameAtAllPlaces: SUCCESS 
ValidateAddedCodeAreRelatedToDuplicateCodeFixing: SUCCESS 
ValidateVariablesUsedInsideExtractedCodeAreAccessible: SUCCESS 
ValidateMethodCanBeStatic: SUCCESS

File: test/src/test/java/hudson/diagnosis/TooManyJobsButNoViewTest.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package hudson.diagnosis;
 
+import org.jvnet.hudson.test.JenkinsRule.WebClient; // CAP AL
```
Comment (audalykh) [inline test/src/test/java/hudson/diagnosis/TooManyJobsButNoViewTest.java:3]: Unused import statement

**Comment by audalykh** [comment]
trilogy-group/CAP-InsightPlugins#1378

**Comment by audalykh** [comment]
Unnecessary added import
trilogy-group/CAP-InsightPlugins#1378

**Pull request closed without merging**